spec/requests/beta/stories_controller_spec.rb

```diff
@@ -0,0 +1,50 @@
+require 'rails_helper'
+
+RSpec.describe 'Ordering stories', type: :request do
+  let(:user) { create :user, :with_team }
+  let(:project) do
+    create(:project, name: 'Test Project', users: [user], teams: [user.teams.first])
+  end
+
+  context 'when moving story in the same state' do
+    let!(:bug) { create(:story, :active, project: project, requested_by: user, new_position: 1) }
+    let!(:chore) { create(:story, :active, project: project, requested_by: user, new_position: 2) }
+    let!(:feature) { create(:story, :active, project: project, requested_by: user, new_position: 3) }
+
+    before do
+      sign_in user
+      PunditContext.new(user.teams.first, user, current_project: project)
```

Comment:
Do we need to instantiate this `PunditContext` here?

Reply:
Removed. After add `config.include Devise::Test::IntegrationHelpers, type: :request` it's no longer accurate